components/shared/CollectionDeposit.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <div>
+    <i18n path="tooltip.deposit" tag="p">
+      <template #amount>
+        <strong>{{ requiredAmount }}</strong>
+      </template>
+      <template #token>
+        <strong>{{ token }}</strong>
+      </template>
+    </i18n>
+    <a v-safe-href="learnMoreLink" class="learn-more">{{
+      $t('helper.learnMore')
+    }}</a>
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  COLLECTION_DEPOSIT_BSX,
+  COLLECTION_DEPOSIT_KSM,
+} from '~~/utils/constants'
+
+const { urlPrefix } = usePrefix()
+
+const token = urlPrefix.value.toUpperCase()
+const requiredAmount =
+  token === 'KSM' ? COLLECTION_DEPOSIT_KSM : COLLECTION_DEPOSIT_BSX
```

Comment:
check out assets page

Reply:
[/bsx](https://kodadot.xyz/bsx/assets/) 3 tokens and [/snek](https://kodadot.xyz/snek/assets/) shows 2. Which one is correct?

Reply:
the one that you've set ie https://beta.kodadot.xyz/bsx/assets/